### ISSUE TITLE
[Feature] rc2nix support backup system-monitor applet

### ISSUE THREAD

**Issue by luochen1990** [nix-community/plasma-manager#241]
Related file: `plasma-org.kde.plasma.desktop-appletsrc`

**Comment by magnouvean** [comment]
I agree this would be nice, but it would be a lot of work due to the way that `plasma-org.kde.plasma.desktop-appletsrc` is structured. `rc2nix`-s intended use-case since `overrideConfig` was introduced is simply to generate a quick configuration to get up-and-running, and as such it has become a less and less important piece in `plasma-manager`. I welcome the suggestion though, and if someone wants to implement feel free to submit a pr.

**Comment by luochen1990** [comment]
I think `rc2nix` is still very important, since sometimes I don't know how to find the name of some option, but if I change it via GUI, and use `rc2nix` to diff with previous version, I can find it out.   So I'm deeply depends on `rc2nix` to use plasma-manger.

**Comment by luochen1990** [comment]
> `rc2nix`-s intended use-case since `overrideConfig` was introduced is simply to generate a quick configuration to get up-and-running, and as such it has become a less and less important piece in `plasma-manager`

I think rc2nix is still very important, since sometimes I don't know how to find the name of some option, but if I change it via GUI, and use rc2nix to diff with previous version, I can find it out. So I'm deeply depends on rc2nix to use plasma-manger.
